wiki/Performance_Points/en.md

```diff
@@ -1,156 +1,82 @@
-                                                                          -   Length of the beatmap in hit objects
-                                                                          -   Combo and amount of Miss.
-                                                                          -   Accuracy {small margin}
-                                                                          -   [Hidden](Hidden "wikilink") and/or [Flashlight](Flashlight "wikilink") is active.                  |                                                              |                                                                                                                 |                                                                                                                                                                                |
-| ![](mania.gif "fig:mania.gif") [osu!mania](osu!mania "wikilink")       |                                                                                                       |                                                              | -   **[Overall difficulty (OD)](Song_Setup#Overall_Difficulty "wikilink")** + [Mods](Game_Modifiers "wikilink") 
-                                                                                                                                                                                                                                                 -   Length of the beatmap in hit objects
-                                                                                                                                                                                                                                                 -   Accuracy                                                                                                     | -   **Strain difficulty** + [Mods](Game_Modifiers "wikilink"), including key-mods and currently excluding [Double Time](Double_Time "wikilink") (problems with scoring system)
-                                                                                                                                                                                                                                                                                                                                                                   -   Length of the beatmap in hit objects
-                                                                                                                                                                                                                                                                                                                                                                   -   Score achieved relative to the maximum possible score (Scaling not linear)                                                                                                  |
 
-'''Note:- '''
+FAQ
```

Comment:
wrong heading level; use level 2 (currently using level 1)

---

you'll also need to increase this section's children's heading levels by 1  too